status/tests/test_status.py

```diff
@@ -0,0 +1,80 @@
+"""
+Tests for the status module.
+"""
+from __future__ import unicode_literals
+
+from copy import deepcopy
+import json
+import logging
+
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.test import Client
+from django.test.testcases import TestCase
+
+log = logging.getLogger(__name__)
+
+
+class TestStatus(TestCase):
+    """Test output of status page."""
+
+    def setUp(self):
+        """Set up client."""
```

Comment:
Pleas add a docstring as to what you are doing here, and maybe a note saying, this saves and restores settings x,y,z in the class docstring so people know what is ok to mess with and what isn't later on.